packages/forms/stories/TextInput.stories.tsx

```diff
@@ -0,0 +1,72 @@
+import { css, cx } from "emotion";
+import * as React from "react";
+import { storiesOf } from "@storybook/react";
+import { withReadme } from "storybook-readme";
+import { TextInput, TextInputAppearance } from "../index";
+const readme = require("../README.md");
+
+const defaultDisplayItem = css`
+  width: 160px;
+`;
+
+storiesOf("Forms/TextInput", module)
+  .addDecorator(withReadme([readme]))
+  .addWithInfo("default", () => (
+    <div>
+      <TextInput
```

Comment:
These don't have an `id` prop passed, but it looks like it's required in your types. Why aren't we getting an error? 🤔

Reply:
it is showing me a warning in my IDE, but not an error. I'll update the stories.